red/api/editor/index.js

```diff
@@ -60,6 +60,11 @@ module.exports = {
                     }
                 });
             }
+            if (settings.editorOptions) {
```

Comment:
Suggest changing `editorOptions` to `httpServerOptions`. GitHub now lets me propose code changes directly, but only one line at a time. So whilst I did do that for the settings file, I'm doing that here as it needs changing in a few places.